extension/src/shared/reearth/layers/story.ts

```diff
@@ -0,0 +1,49 @@
+import { FC, useMemo } from "react";
+
+import { composeIdentifier } from "../../../prototypes/cesium-helpers";
+import { StoryCapture } from "../../layerContainers/story";
+import { useLayer } from "../hooks";
+import { Data, LayerAppearanceTypes } from "../types";
+
+export type StoryAppearance = Partial<Pick<LayerAppearanceTypes, "marker">>;
+
+export type StoryProps = {
+  capture: StoryCapture;
+  appearances: StoryAppearance;
+  visible?: boolean;
+  onLoad?: (layerId: string) => void;
+};
+
+export const STORY_MARKER_ID_PROPERTY = "storyCaptureID";
+
+export const StoryLayer: FC<StoryProps> = ({ capture, appearances, visible, onLoad }) => {
+  const data: Data = useMemo(() => {
+    const objectId = composeIdentifier({
+      type: "Story",
+      key: capture.id,
+    });
+
+    return {
+      type: "geojson",
+      value: {
+        type: "Feature",
+        properties: {
+          [STORY_MARKER_ID_PROPERTY]: objectId,
```

Comment:
Is it working? Should we generate the id for each marker...?

Reply:
Yes this `StoryLayer` is actually marker layer.
So each capture will generate a objectId and been used in two reearth layers (one for balloon icon and one for camera icon). Maybe i can refactor some to make it looks more clear but basically it's okey i think.